app/bundles/LeadBundle/Form/Type/ListType.php

```diff
@@ -57,6 +58,12 @@ public function __construct(MauticFactory $factory)
             $this->listChoices[$list['id']] = $list['name'];
         }
 
+        $emails                = $listModel->getUserEmails();
+        $this->emailChoices     = array();
+        foreach ($emails as $email) {
```

Comment:
I don't think it'll return null unless I overlooked something?  `$listModel->getUserEmails()` calls `EmailRepository::getEmailList()` which returns `$q->getQuery()->getArrayResult();` which should always be an array even if empty.